tests/rptest/clients/kubectl.py

```diff
@@ -361,7 +381,23 @@ def _setup_tbot(self):
             f'--token={self._tp_token}', '--certificate-ttl=6h',
             '--renewal-interval=6h', '--oneshot'
         ]
-        self._local_cmd(cmd)
+        # Log the full command to be executed
+        self._redpanda.logger.debug(f"Running tbot command: {cmd}")
+
+        try:
+            self._local_cmd(cmd)
+        except subprocess.CalledProcessError as e:
```

Comment:
In case of error, _local_cmd func will already log stdout and stderr. I suggest we remove lines 390-393 to prevent double logging

Reply:
Good catch, thank you

Reply:
Removed